code/modules/mob/living/carbon/superior_animal/roach/attack.dm

```diff
@@ -9,9 +9,11 @@
 			if(istype(howdoitameahorseinminecraft))
 				if(try_tame(H, howdoitameahorseinminecraft))
 					return FALSE //If they manage to tame the roach, stop the attack
-		if(istype(L) && !L.weakened && prob(15) && src.density) // Syzygy code to limit knockdown to only dense roaches.
+		if(istype(L) && !L.weakened && ((prob(13) && src.density) || prob(2))) // Syzygy code to limit knockdown to only dense roaches.
```

Comment:
You might want to replace this comment and comment the other changes here, since it's non-modular, and that comment is now out of date and wrong with this change.